.github/workflows/rust.yml

```diff
@@ -62,6 +62,7 @@ jobs:
         uses: actions-rs/cargo@v1
         with:
           command: check
+          args: -Z msrv-policy
```

Comment:
Just to clarify, this is a nightly feature / experiment. Until broader adoption of `rust-version`, it may not be able to resolve dependencies for the `rust-version` declared in your `Cargo.toml`.

You may want to verify that it builds for the given MSRV (_eg: `cargo +1.60.0 check` or equivalent_), as newer toolchains may install the dependencies without issue.

---

`ordered-multimap = "0.4"` is an example where `cargo +1.56.1 check` would fail.
- Without `-Z msrv-policy` it would have resolved `ordered-multimap 0.4.0` instead of `ordered-multimap 0.4.3` due to the newer releases of that crate resolving dependencies (_such as `ahash 0.7.6` and newer_) utilizing `Cargo.toml` features that were introduced in Rust `1.60.0`.
- Earlier toolchains consider it invalid `Cargo.toml` and resolve an earlier crate release within the semver range instead.

Thus to properly verify, after nightly `-Z msrv-policy` you'd probably want to test against the expected Rust toolchain release you're declaring support for.

---

As explained above, you could test such without `-Z msrv-policy`. When that feature works, it's to prevent resolving crates within a semver range that have declared a higher `rust-version`.
- Without `-Z msrv-policy` would fail (potentially useful signal)
- With a working `-Z msrv-policy` failure is avoided and `check` should be successful on the MSRV toolchain.
  - Otherwise requires `cargo update --package --precise version_here` to pin the last version of a crate that was compatible.
  - Alternatively, it's a signal that you raise your own crate MSRV and update `rust-version`. If it's from an upstream dependency not using `rust-version` earlier releases of `ahash` would likewise fail without the `Cargo.lock` management.
  - Sometimes upstream crates relax / lower their MSRV, as recently happened with `zerocopy` that motivated this PR. If that happens and you raised `rust-version` already, you can still release again with a lowered `rust-version` too if preferred (_`-Z msrv-polciy` resolves semver normally, then works backwards AFAIK_).